data-imports/import-pluto.py

```diff
@@ -229,7 +229,7 @@ def import_csv(args):
 
         sys.stdout.write("\rConcatenating Pluto boro csvs....")
         pluto_csv_files_dir = os.path.join(csv_dir, "BORO_zip_files_csv")
-        pandas_concat_csv(pluto_csv_files_dir, csv_file)
+        simple_concat_csv(pluto_csv_files_dir, csv_file)
```

Comment:
Hmm, simply concat'ing the pluto files together resulted in some oddities. If I recall correctly, the columns don't line up across pluto files, and using pandas to concat resolved it. @jpfreeley and @mcleanalec, does that sound right?

After you re-imported, did you see any issues?